tests/integration_tests/data_steward/validation/ehr_union_test.py

```diff
@@ -447,8 +459,17 @@ def test_ehr_person_to_observation(self):
         self.assertEqual(len(expected), len(actual))
         self.assertCountEqual(expected, actual)
```

Comment:
Is the second assertion same as the previous one?

Reply:
I think it was part of a change we made during migration. One of them was testing for the individual items while the other was testing for the keys. I didn't make this change for this ticket but I could remove one of them for now

Reply:
Thanks for the explanation. Sorry this is totally not within the scope! I was just trying to run the test from my machine and step through the code, that's when I saw these two assertions. I thought it might be a good idea to remove one if they were checking the same thing.

Reply:
I'll remove the `assertEqual` since `assertCountEqual` handles list of dictionaries